rts/Rml/SolLua/bind/DataModel.cpp

```diff
@@ -62,17 +62,24 @@ namespace Rml::SolLua
 		 * You can access fields by using normal indexing, but for the moment, the only keys that work are strings. Any index done this way will automatically trigger a rerender.
 		 * If you need to index any tables or subtables by anything not a string, you will need to use the underlying table, gotten with `__GetTable`. This will not trigger a rerender.
 		 * To trigger a rerender manually, use `_SetDirty`, passing in the name of the top-level entry in your model table that you edited.
-		 * @class RmlUi.SolLuaDataModel<T> : {__GetTable:fun():T}
+		 * @class RmlUi.SolLuaDataModel<T>
 		 */
 
-		 /***
-		  * Set a table property dirty to trigger a rerender
-		  * @function RmlUi.SolLuaDataModel:__SetDirty
-		  * @param property string
-		  */
+		/***
+		 * Get the underlying table. Note: Implemented in next release as of May 9 2025. See [PR](https://github.com/beyond-all-reason/RecoilEngine/pull/2241)
```

Comment:
This kind of thing makes sense as a note, because we're constantly updating the library. But long-term the plan is to tag the library when we do a release, so consumers can target a specific release tag. That would mean that this note would only be visible when it's already available (i.e. "next release" would be untrue).

For now I've set BAR to update the library on every release because the vast majority of changes are fixes + improvements to the library, and only a small minority are new API calls.

I'd probably say this kind of comment is not needed because it's liable to get stale.

Ultimately the plan is to do versioned docs, so that will handle this.

Open to discussion though.